### DISCUSSION TITLE
How to set up (fix) the home page of my Quarto website

### DISCUSSION THREAD

**Discussion by LuisLauM** [quarto-dev/quarto-cli#9430] [Q&A] [answered]
### Description

Hello, hello.

I am creating my personal website using Quarto and Github pages. Everything is going well for the moment, however, since I decided to include a section (called Blog) using listings, now every time I try to enter my website (waiting for it to show me the Home), it shows an app I made using dashboard and that is inside Blog/. 

I would like to know if this can be fixed from Quarto or if it is an external problem (from Github pages maybe). The link to my website is the following, as you can see the link does not make direct reference to the app inside Blog/ and nevertheless when one clicks on it is where it automatically redirects to:

[https://luislaum.github.io/](https://luislaum.github.io/)

Thank you!

**Comment by mcanouil** [comment, marked as answer]
Set an index page which you don't have.
Rename you "home.qmd" as index.qmd.

**Comment by LuisLauM** [comment]
Excellent! I didn't know about to set up the main file with index. Thank you so much.
